config/game.ini

```diff
@@ -121,10 +121,13 @@ no_pickup=5,530,240,65
 repair_needed=1000,0,280,280
 wp_act_roi=42,66,365,53
 stash_btn_roi=20,52,412,53
+hero_stash_roi=35,92,378,378
```

Comment:
`hero_stash_roi` and `vendor_stash` should be equivalent. I'd pick one rather than have two. Personally I'd consider making generic names like `left_inventory` for left panel and `right_inventory` for right panel inventory or something that can be used in vendor routines and stashing routines and change existing names accordingly. Just my 2 cents :)